streams/src/main/java/org/apache/kafka/streams/state/internals/SegmentedBytesStore.java

```diff
@@ -114,7 +125,18 @@
          * @param to            ending time range
          * @return
          */
-        HasNextCondition hasNextCondition(final Bytes binaryKey, long from, long to);
+        HasNextCondition hasNextCondition(final Bytes binaryKey, final long from, final long to);
```

Comment:
I'm happy to use a single method for now. In the essence of time. If there is a clear need for both, I'll do a follow-up PR.